bedrock/products/templates/products/vpn/download.html

```diff
@@ -37,121 +39,219 @@
+          </div>
+          <div class="platform-body">
+            <h2>{{ ftl('vpn-download-download-for-mac') }}</h2>
+            <p class="current-platform-lede">{{ ftl('vpn-download-based-on-your') }}</p>
+            <p>{{ ftl('vpn-download-version-requirements', version='10.14') }}</p>
+            <a class="mzp-c-button" href="{{ mac_download_url }}" data-cta-type="button" data-cta-text="VPN Download (macOS)" class="platform-download-link">
+            {{ ftl('vpn-download-get-mozilla-vpn') }}
+            </a>
+          </div>
+        </div>
+        <!-- Linux -->
+        <div class="primary-platform linux">
+          <div class="platform-image">
+            <img src="{{ static('img/products/vpn/download/platform-linux.svg') }}" alt="">
+          </div>
+          <div class="platform-body">
+            <h2>{{ ftl('vpn-download-download-for-linux') }}</h2>
+            <p class="current-platform-lede">{{ ftl('vpn-download-based-on-your') }}</p>
+            <p>{{ ftl('vpn-download-for-linux-requirements', version='18.04') }}</p>
+            <a class="mzp-c-button" href="{{ linux_download_url }}" data-cta-type="button" data-cta-text="VPN Download (Linux)" class="platform-download-link">
+            {{ ftl('vpn-download-get-mozilla-vpn') }}
+            </a>
+          </div>
+        </div>
+        <!-- iOS -->
+        <div class="primary-platform ios">
+          <div class="platform-image">
+            <img src="{{ static('img/products/vpn/download/platform-ios.svg') }}" alt="">
+          </div>
+          <div class="platform-body">
+            <h2>{{ ftl('vpn-download-download-for-ios') }}</h2>
+            <p class="current-platform-lede">{{ ftl('vpn-download-based-on-your') }}</p>
+            <p>{{ ftl('vpn-download-version-requirements', version='13.0') }}</p>
+            <a href="{{ ios_download_url }}" data-cta-type="button" data-cta-text="VPN Download (iOS)" class="platform-download-link">
+              <img src="{{ static('img/products/vpn/download/apple-app-store-badge.svg') }}" alt=" {{ ftl('vpn-download-get-mozilla-vpn') }}">
+            </a>
+          </div>
+        </div>
+        <!-- Android -->
+        <div class="primary-platform android">
+          <div class="platform-image">
+            <img src="{{ static('img/products/vpn/download/platform-android.svg') }}" alt="">
+          </div>
+          <div class="platform-body">
+            <h2>{{ ftl('vpn-download-download-for-android') }}</h2>
+            <p>{{ ftl('vpn-download-version-requirements', version='8.0') }}</p>
+            <p>{{ ftl('vpn-download-version-requirements', version='13.0') }}</p>
+            <a class="platform-download-link" href="{{ android_download_url }}" data-cta-type="button" data-cta-text="VPN Download (Android)" target="_blank" rel="noopener noreferrer">
+              <img src="{{ static('img/products/vpn/download/google-play-badge.png') }}" alt=" {{ ftl('vpn-download-get-mozilla-vpn') }}">
```

Comment:
Suggestion (non-blocking): should these mobile store badges have a set `width` and `height` defined? The Android one in particular shows up pretty large?

![image](https://user-images.githubusercontent.com/400117/231710228-ca6cd337-b848-424d-bf28-d598cbd6398f.png)